src/multi_processing/pool_tasks.py

```diff
@@ -1,120 +1,157 @@
-""" Pool Manager for both Generation and Writing tasks. Two methods are
-responsible for each, a catch-all generate/write method which configures
-the Pool for jobs to be executed on. Additionally included are methods the
-pools use for each job in their respective job lists to perform the actual
-generation/writing tasks.
+""" Pool Manager functionality for both create and write parent processes.
+Two methods are responsible for each, run_create_jobs and run_write_jobs, which
+run a batch of jobs over a configured pool of child processes. Helper methods
+are also provided to assist in running the jobs.
 """
```

Comment:
Unclear what the above comments mean.  Are run_create_jobs and run_write_jobs the "Two methods" or the "each"?